### PR TITLE
Front end mvc

### PR DESCRIPTION
**Pull request by nchurchill**
I have added in a schedule to the schedule page. I added in lists on the schedule page with the courses on cards that have the ability to be added to the aforementioned schedule. I put the user page onto two separate tabs rather than all of it being on one screen.

### CODE REVIEW

File: src/Athena/Athena.csproj [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
+    <NoWin32Manifest>true</NoWin32Manifest>
```
Comment (nlowe) [inline src/Athena/Athena.csproj]: Is this a visual studio ism or did you have to add this to fix something?

File: src/Athena/Content/css/site.css [hunk 2]
```diff
@@ -5,8 +5,50 @@
...
+        display: none !important;
+    }
+
+    .material-tooltip {
+        display: none !important;
+    }
+
+   .card-panel, .card{
+        -webkit-box-shadow: 0 0px 0px 0 rgba(0, 0, 0, 0), 0 0px 0px 0 rgba(0, 0, 0, 0), 0 0px 0px 0px rgba(0, 0, 0, 0);
+        box-shadow: 0 0px 0px 0 rgba(0, 0, 0, 0), 0 0px 0px 0 rgba(0, 0, 0, 0), 0 0px 0px 0px rgba(0, 0, 0, 0);
```
Comment (nlowe) [inline src/Athena/Content/css/site.css]: What's all these magic `0`'s for?

File: src/Athena/Content/css/site.css [hunk 3]
```diff
@@ -78,16 +116,94 @@ main {
...
     position: absolute;
     width: 70%;
     left: 0;
+}
+
+.cal-card-content{
+    background-color:lightgray;
+}
+
+user-tabs {
```
Comment (nlowe) [inline src/Athena/Content/css/site.css]: This isn't a valid element name. Did you mean to ref the `user-tabs` id (`#user-tabs`) or class (`.user-tabs`)?

File: src/Athena/Content/css/site.css [hunk 4]
```diff
@@ -78,16 +116,94 @@ main {
...
+    margin-right:80px;
+}
+
+.courseTable tbody {
+    display: block;
+    height: 200px;
+    overflow: auto;
+}
+
+    .courseTable thead, .courseTable tbody tr {
```
Comment (nlowe) [inline src/Athena/Content/css/site.css]: Remove extra indentation here

File: src/Athena/Views/Home/Add_Classes.cshtml [hunk 5]
```diff
@@ -2,29 +2,90 @@
...
+            <h1 class="btn col s6 offset-s3">Completed Courses</h1>
+        </div>
+        <div>
+            <p>Second Section Content</p>
+        </div>
     </div>
+    <br />
     <div class="row">
-        <h1 class="btn col s6 offset-s3">All Courses</h1>
+        <a href="User_Page" class="btn col s6 offset-s3">Finish</a>
```
Comment (nlowe) [inline src/Athena/Views/Home/Add_Classes.cshtml]: Use `@Url.Action(...)` to link to other pages.

File: src/Athena/Views/Home/Home.cshtml [hunk 6]
```diff
@@ -9,6 +9,7 @@
     <p>
         This tool was designed by students for students, to allow for a more streamlined approach to allow students to schedule their course work for each semester and allow them to track thier progression through their degree.
     </p>
+    <p id="test"></p>
```
Comment (nlowe) [inline src/Athena/Views/Home/Home.cshtml]: ?

Comment (nchurchill) [reply src/Athena/Views/Home/Home.cshtml]: gone

File: src/Athena/Content/css/site.css (hunk 4 above)
Comment (nchurchill) [reply src/Athena/Content/css/site.css]: done

File: src/Athena/Content/css/site.css (hunk 3 above)
Comment (nchurchill) [reply src/Athena/Content/css/site.css]: period added

File: src/Athena/Views/Home/Add_Classes.cshtml (hunk 5 above)
Comment (nchurchill) [reply src/Athena/Views/Home/Add_Classes.cshtml]: changed

File: src/Athena/Content/css/site.css (hunk 2 above)
Comment (nchurchill) [reply src/Athena/Content/css/site.css]: This is to print

**Pull request merged**